doc/explanations/min-utxo-alonzo.rst

```diff
@@ -0,0 +1,99 @@
+Min-Ada-Value Calculation in Alonzo
+===================================
+
+The calculation
+#################
+
+There is no longer a ``minUTxOValue`` protocol parameter in Alonzo, it was
+deprecated at the end of the Mary era. However, the requirement that each
+UTxO must contain an amount of ada that depends on the size of the entry.
+The size-dependent minimum ada amount in a UTxO is still referred to as the min-ada-value, and
+is now calculated using the Alonzo parameter ``coinsPerUTxOWord``.
+
+The minimum ada value calculation relies on the ``size`` function for determining
+the size of a token bundle or a lovelace value, which is described in
+the Mary era min-value document.
+
+The formula for the min-ada-value calculation, for a UTxO containing an output ``txout`, is:
+
+``utxoEntrySize (txout) * coinsPerUTxOWord``
+
+where
+
+``utxoEntrySize (txout) = utxoEntrySizeWithoutVal + size (v) + dataHashSize (dh)``
+
+The ``v`` variable is the token bundle inside the ``txout`` output, and
+``dh`` is the hash of the datum that is contained in the output. If there is
+no datum, the hash is represented by ``Nothing``, and takes up ``0`` space.
```

Comment:
This explains one branch of `dataHashSize` but not the other (which is given by the table below). Either both should be explained here, or none (and referring to the table below)

Reply:
the size calculation is explained in the Mary document (and should stay there for non-duplication)

Reply:
I'm not talking about `size`, but about `dataHashSize`. This paragraph gives half of its definition, and it should either be all of it, or none of it.